compute/wps/tasks/cdat.py

```diff
@@ -125,6 +125,8 @@ def base_retrieve(self, attrs, keys, operation, var_name, base_units, output_pat
 
     start = self.get_now()
 
+    os.makedirs(os.path.dirname(output_path))
```

Comment:
you might want this in a try/except statement. I think it fails if directory already exists.

Reply:
Or you can only make it if doesn't exist.

Reply:
Added try/except, better safe than sorry.